site/content/docs/user/quick-start.md

```diff
@@ -218,13 +218,13 @@ context name `kind` and delete that cluster.
 
 ## Loading an Image Into Your Cluster
 
-Docker images can be loaded into your cluster nodes with:
+You can also load more than one image at once by passing multiple image names:
```

Comment:
This is now missing the single image example. Looks like it was added below. That section should move up and replace this section's content here.

Reply:
With the correction for not passing the explicit cluster name and just keeping that detail in the existing note.